packages/core/src/env-set/oidc.ts

```diff
@@ -33,14 +33,16 @@ const readPrivateKey = async (): Promise<string> => {
       throw error;
     }
 
-    const answer = await inquirer.prompt({
-      type: 'confirm',
-      name: 'confirm',
-      message: `No private key found in env \`OIDC_PRIVATE_KEY\` nor \`${privateKeyPath}\`, would you like to generate a new one?`,
-    });
+    if (!allYes) {
+      const answer = await inquirer.prompt({
+        type: 'confirm',
+        name: 'confirm',
+        message: `No private key found in env \`OIDC_PRIVATE_KEY\` nor \`${privateKeyPath}\`, would you like to generate a new one?`,
+      });
 
-    if (!answer.confirm) {
-      throw error;
+      if (!answer.confirm) {
+        throw error;
+      }
```

Comment:
Seems like this PR affects test coverage. Do we need to handle it?

Reply:
we'll go through them later